lib/errors/arsenalErrors.ts

```diff
@@ -247,7 +247,7 @@ export const InvalidURI: ErrorFormat = {
     description: "Couldn't parse the specified URI.",
 };
 
-export const KeyTooLong: ErrorFormat = {
+export const KeyTooLongError: ErrorFormat = {
```

Comment:
This is breaking change https://github.com/search?q=org%3Ascality+KeyTooLong+path%3A*.js+OR+path%3A*.ts+OR+path%3A*.rst+OR+path%3A*.json&type=code

Reply:
Ok, I'll work with the old version for CS Post Object then